app/controllers/peeps.rb

```diff
@@ -0,0 +1,23 @@
+
+class Chitter < Sinatra::Base
+
+  get '/peeps' do
+    @peeps = Peep.all
+    erb :'peeps/index'
+  end
+
+  get '/peeps/new' do
+    erb :'peeps/new'
+  end
+
+  post '/peeps' do
+    if current_user
+      peep = Peep.create(message: params[:peep], user_id: session[:user_id])
+      redirect '/peeps'
+    else
+      flash.now[:errors] = ['You must be signed in to post a peep']
```

Comment:
Can this code be made more declarative of what it does? At the moment, it seems to reveal its implementation, rather than its intention.